observations/obs_converters/even_sphere/run_fixed_network_daily.csh

```diff
@@ -4,10 +4,13 @@
 # by UCAR, "as is", without charge, subject to all terms of use at
 # http://www.image.ucar.edu/DAReS/DART/DART_download
 
-# loop, calling ./create_fixed_network_seq to create separate
-# files for each time period.  edit the values below to change
-# the dates and intervals.
+# Call ./create_fixed_network_seq to create a separate file for each time period.  
+# Edit the values below to change the dates and intervals.
+# It makes 1 file/day, single time per file.
```

Comment:
maybe mention in this comment that the single time per day is 12Z?  i might assume 0Z otherwise.

Reply:
I changed the comment.
I don't know why 12 UTC was chosen for this.
In the future I think that it is redundant with run_fixed_network_seq.csh 
and could be removed.